optimum/quanto/tensor/qbits/qbits.py

```diff
@@ -55,7 +56,19 @@ def backward(ctx, gO):
 
 class QBitsTensor(QTensor):
     @staticmethod
-    def create(qtype, axis, group_size, size, stride, data, scale, shift, requires_grad=False):
+    def create(
+        qtype,
+        axis,
+        group_size,
+        size,
+        stride,
+        data,
+        scale,
+        shift,
+        activation_qtype: Optional[qtype] = None,
```

Comment:
Thank you, this change stems from the fact that we need to know the activation qtype to choose which tensor subclass to instantiate for weights. As this may happen both for qbytestensor and qbitstensor and as some part of the codebase handles both, I added it here as well.